packages/lab/src/pagination/Paginator.tsx

```diff
@@ -51,47 +46,35 @@ export const Paginator = forwardRef<HTMLDivElement, PaginatorProps>(
 
     const onNextPage = useCallback(() => {
```

Comment:
Do we need useCallback here? The rule is generally if it's being passed to the DOM, it's not needed unless you're using memoisation